tests/php/src/Internal/RestApiUtilTest.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+namespace Automattic\WooCommerce\Tests\Internal;
+
+use Automattic\WooCommerce\Internal\RestApiUtil;
+
+/**
+ * Tests for the RestApiUtil class.
+ * @package Automattic\WooCommerce\Tests\Internal
+ */
+class RestApiUtilTest extends \WC_Unit_Test_Case {
```

Comment:
can we add a test with just `amount` param and no line items to preserve backward compat?

Reply:
Yep, good point.

Reply:
Actually this case is already covered in one of the tests (see `data_for_test_adjust_create_refund_request_parameters_does_not_change_amount_if_invalid_data`)